src/pages/login.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NextPage } from 'next'
+import KakaoButton from '../components/KakaoButton';
+import GithubButton from '../components/GithubButton';
+
+const Login : NextPage<Props> = (props) => {
```

Comment:
```suggestion
const Login : NextPage = () => {
```

아까 리뷰에서 props는 임의로 넣어놓은 값이라서 지금은 필요없을 것 같아요! 나중에 ssr 할때 사용하게 될 것 같습니당